### PR TITLE
Block format

### PR DESCRIPTION
**Pull request by sglanzer-fire**
### This project uses [semver](semver.org), please check the scope of this pr:
- [ ] #patch# - backwards-compatible bug fix
- [x] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change
# CHANGELOG
- Added block format usage

### CODE REVIEW

File: addon/pods/components/frost-tabs/template.hbs [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+        parentHook=hook
+        targetOutlet=targetOutlet
+        selectedTab=selectedTab
+        onChange=onChange
+      )
+    )}}
+  {{else}}
+    {{#each tabs as |tab index|}}
+      <div data-test={{hook (concat hook '-tab') index=index}}>
+        {{component tab parentHook=hook targetOutlet=targetOutlet selectedTab=selectedTab onChange=onChange}}
```
Comment (dafortin) [inline addon/pods/components/frost-tabs/template.hbs:14]: Can you move this on multiple lines like you did above? It's cleaner.

File: tests/dummy/app/pods/demo/block-format/template.hbs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{{!-- BEGIN-SNIPPET overview-content-api }}
```
Comment (dafortin) [inline tests/dummy/app/pods/demo/block-format/template.hbs:1]: typo `overview`

File: tests/dummy/app/pods/demo/block-format/template.hbs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+{{frost-tabs
```
Comment (dafortin) [inline tests/dummy/app/pods/demo/block-format/template.hbs:2]: You might want to show the block format API?

File: tests/dummy/app/pods/demo/block-format/template.hbs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+<div class='addon-demo-api'>
+  <div class='addon-demo-title'>
+    How did it get so late so soon? - Dr. Seuss
+  </div>
+</div>
+<div class='addon-demo-live'>
+  <div class='addon-demo-title'>
+    Live demo
+  </div>
+  <div class='addon-demo-snippet'>
```
Comment (dafortin) [inline tests/dummy/app/pods/demo/block-format/template.hbs:24]: Do you want to show the controller also here?

File: tests/dummy/app/router.js [hunk 5]
```diff
@@ -17,6 +17,7 @@ Router.map(function () {
     this.route('tab')
     // Tests
     this.route('hook')
+    this.route('block-format')
```
Comment (dafortin) [inline tests/dummy/app/router.js:20]: Should probably be in one of the section above and not under test. I tried to make this match with the overview page.

**Comment by sandersky** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-tabs/pull-request/29/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-tabs)

File: tests/dummy/app/router.js (hunk 5 above)
Comment (dafortin) [inline tests/dummy/app/router.js]: You will also need to modify the overview page to see the new demo on the left panel.

**Pull request merged**